### PR TITLE
Poland Update to fix #272 + ReadMe fix for Ireland

### LINKED ISSUE #272: Possible update on solar capacity in PL
Seen in Google Forms:

information about the installed PV in Poland (missing on the website currently) can be found here: https://www.ure.gov.pl/pl/rynki-energii/energia-elektryczna/odnawialne-zrodla-ener/potencjal-krajowy-oze/5753,Moc-zainstalowana-MW.html

### PR DESCRIPTION
**Pull request by ThierryOllivero**
- Update of capacities.json
- Update of ReadMe for Poland
- Fix of ReadMe for Ireland

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,7 +93,12 @@ Each country has a GHG mass flow that depends on neighboring countries. In order
   - Gas: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
   - Hydro: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
   - Wind: [ieawind.org](http://www.ieawind.org/countries/norway.html)  
-- Poland: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
+- Poland: 
+  - Biomass : [Ure.gov.pl](https://www.ure.gov.pl/pl/rynki-energii/energia-elektryczna/odnawialne-zrodla-ener/potencjal-krajowy-oze/5753,Moc-zainstalowana-MW.html)
```
Comment (corradio) [inline README.md]: There's a space that shouldn't be there (before the `:`)

File: README.md [hunk 2]
```diff
@@ -93,7 +93,12 @@ Each country has a GHG mass flow that depends on neighboring countries. In order
...
+  - Hydro : [Ure.gov.pl](https://www.ure.gov.pl/pl/rynki-energii/energia-elektryczna/odnawialne-zrodla-ener/potencjal-krajowy-oze/5753,Moc-zainstalowana-MW.html)
+  - Solar : [Ure.gov.pl](https://www.ure.gov.pl/pl/rynki-energii/energia-elektryczna/odnawialne-zrodla-ener/potencjal-krajowy-oze/5753,Moc-zainstalowana-MW.html)
+  - Wind : [Ure.gov.pl](https://www.ure.gov.pl/pl/rynki-energii/energia-elektryczna/odnawialne-zrodla-ener/potencjal-krajowy-oze/5753,Moc-zainstalowana-MW.html)
```
Comment (corradio) [inline README.md]: Weird, for some reason the `-` is not formatted correctly. In the preview it also has bad formatting :(

**Comment by corradio** [comment]
@brunolajoie can you quickly review? Just check that the order of magnitudes of the numbers seem ok. It should only take 30s :)

**Comment by brunolajoie** [comment]
The IRENA provides a reference database of installed renewable capacities per country here, which stops in 2015. http://resourceirena.irena.org/gateway/dashboard/?topic=4&subTopic=16
 
IRENA data differs from the pdf provided by the polish regulator, especially for hydro and biogaz. 
I suggest as a general good practice to follow the polish regulator data because it is probably closer to the source. However, hydro capacity has been significantly reduced from 2GW+ to 900MW in this new pull request, so I would like to double check polish historical production over the last few months to see if we don't get capacity factor above 100%. -->@oli could you print me the time series of polish produciton per source over the last few months? thank you

**Comment by corradio** [comment]
@brunolajoie :
image
and because I like you, I also give you the max values over that period:
image

**Comment by ThierryOllivero** [comment]
So, max(hydro) is superior to capacity announced by the regulator. Can it be possible? How is the capacity calculated? 

I know that for solar, you can get more than 100% of capacity, since capacity in Wp is defined with an irradiation of 1000W/m². If you have more irradiation, you go over nominal capacity. Could it be the case for Poland?

**Comment by brunolajoie** [comment]
For Hydro, I might have the answer: Regolator's 987MW is probably hydropower only, excluding Pumped Hydro Storage capacity, which accounts for 1.7GW as per both IRENA and ENTSOE in agreement. Taking ENSTOE data should be correct.

For biomass, polish regulator figure is much higher than ENTSOE or IRENA data, and max output as per oli datase. I suspect that the regulator's 1273MW is not only MWelectric but includes some heat. I recommand using ENTSOE data.

Ultimately, I think ENSTOE data is more reliable for our purpose: electricity only. Why don't we just use their data for capacity installed?

**Comment by ThierryOllivero** [comment]
Because when we use data from ENTSOE, we have sometimes Production > Capacity. Example: Solar in PL which gave issue #272 and this PR :)

**Comment by brunolajoie** [comment]
Do you have an exemple when solar output > ENSTOEcapacity? Because this pull request changed solar from 77 to 92MW and i don't think its a big difference. Issue comes probalby more from entsoe solar output data

**Comment by brunolajoie** [comment]
@corradio @ThierryOllivero  So I suggest to keep using ENSTSOE 2016 capacity data.

**Comment by ThierryOllivero** [comment]
Agreed.

**Pull request closed without merging**